src/pages/health-declaration/health-declaration.ts

```diff
@@ -131,18 +131,21 @@ export class HealthDeclarationPage extends BasePageComponent {
       candidateName$: this.store$.pipe(
         select(getTests),
         select(getCurrentTest),
+        select(getJournalData),
```

Comment:
Worth thinking about extracting `journalData$` for simplified code

Reply:
In the future yes - for now I would say it's useful there to let people know there is another layer in the app

Reply:
Could have reused the `currentTest$` observable at least